pkgs/development/python-modules/nanobind/default.nix

```diff
@@ -1,31 +1,31 @@
 {
   lib,
   buildPythonPackage,
-  pythonOlder,
   fetchFromGitHub,
+  pytestCheckHook,
+  pythonOlder,
   cmake,
   eigen,
+  jax,
+  jaxlib,
   ninja,
-  scikit-build,
-  pytestCheckHook,
   numpy,
+  pathspec,
+  scikit-build-core,
   scipy,
+  tensorflow-bin,
   torch,
-  jax,
-  jaxlib,
-  tensorflow,
-  setuptools,
 }:
```

Comment:
The inputs should be sorted by order of apparition, not alphabetically.
[Example from the documentation](https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-function)

Reply:
done.